compiler/p/codegen/ControlFlowEvaluator.cpp

```diff
@@ -3836,62 +3841,65 @@ static TR::Register *generateMaxMin(TR::Node *node, TR::CodeGenerator *cg, bool
          }
       if (two_reg)
          {
-         generateTrg1Src1Instruction(cg, move_op, node, trgReg->getLowOrder(), reg->getLowOrder());
-         generateTrg1Src1Instruction(cg, move_op, node, trgReg->getHighOrder(), reg->getHighOrder());
+         generateTrg1Src1Instruction(cg, move_op, node, trgReg->getLowOrder(), src1Reg->getLowOrder());
+         generateTrg1Src1Instruction(cg, move_op, node, trgReg->getHighOrder(), src1Reg->getHighOrder());
          }
       else
          {
-         generateTrg1Src1Instruction(cg, move_op, node, trgReg, reg);
+         generateTrg1Src1Instruction(cg, move_op, node, trgReg, src1Reg);
          }
       }
 
-   int n = node->getNumChildren();
-   for (int i = 1; i < n; i++)
+   TR::Register *condReg = cg->allocateRegister(TR_CCR);
+   TR::LabelSymbol *label = generateLabelSymbol(cg);
```

Comment:
It looks like the internal control flow here was never marked correctly. Please add a label with `setStartInternalControlFlow()` called on it before the first conditional branch and then call `setEndInternalControlFlow()` on the end label.

Reply:
Fixed in d04f5d9